data/json/items/gun/30carbine

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "m1carbine",
+    "copy-from": "rifle_semi",
+    "looks_like": "garand",
+    "type": "GUN",
+    "name": { "str": "M1 carbine" },
+    "description": "A light rifle developed during the Second World War to provide a less cumbersome weapon for soldiers that didn't need the firepower of an M1 Garand. Firing the .30 Carbine cartridge, it provides more firepower than a pistol and is easier to carry than most other rifles.",
+    "weight": "2400 g",
+    "volume": "2800 ml",
+    "longest_side": "900 mm",
+    "price": 80000,
+    "price_postapoc": 3500,
+    "to_hit": -1,
+    "bashing": 9,
+    "material": [ "steel", "wood" ],
+    "symbol": "(",
+    "color": "brown",
+    "ammo": [ "30carbine" ],
+    "dispersion": 200,
+    "durability": 8,
+    "min_cycle_recoil": 900,
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "barrel", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail", 1 ],
```

Comment:
for stuff like rail it and grip those should probably be 
```rail mount```
```grip mount```
etc.

since the gun doesn't have modern rails or mount point so you would need to do permanent aftermarket changes to attach stuff like lasers and grips. 

I was going to recommend you look at the Garand but it seems to also be missing the mount entries.

The m1903 has what I'm talking about.

Reply:
Yup, I copied it from the Garand, thinking it was up to snuff. Funny.